code/game/objects/items/stacks/sheets/leather.dm

```diff
@@ -178,7 +178,7 @@ GLOBAL_LIST_INIT(sinew_recipes, list (
 
 /obj/item/stack/sheet/animalhide/goliath_hide
 	name = "goliath hide plates"
-	desc = "Pieces of a goliath's rocky hide, these might be able to make your suit a bit more durable to attack from the local fauna."
+	desc = "Pieces of a goliath's rocky hide, these might be able to make your miner suit, miner plasmaman helmet, miner borg or Ripley class exosuits a bit more durable to attack from the local fauna."
```

Comment:
Too much "miner" repetition, see if you can rewrite in a better way

Reply:
okay!